gulpfile.js

```diff
@@ -26,6 +27,8 @@ const PACK_SRC = "src/packs";
 const BUILD_DIR = "build";
 const DIST_DIR = "dist";
 const CSS_DEST = path.join(BUILD_DIR, "css");
+const FOUNDRY_DIR = ".local/share/FoundryVTT/Data/systems";
```

Comment:
There's no guarantee this will be the users Foundry data directory. For example, I have moved mine to a Dropbox folder so that it is always backed up.

Reply:
Yeah, this requires the developer to specify exactly where their install is located which is a problem. This is also just the default directory for Linux, there are other locations for Mac and Windows.

I'll remove the `gulp deploy` change from this PR which should address the requested change.

Reply:
Yeah, I don't think it adds a great deal tbh. Let me know when that's done and I'll look at merging the PR